src/soc/aspeed/ast25x0/Makefile

```diff
@@ -0,0 +1,4 @@
+OREBOOT=$(abspath $(CURDIR)/../../../../)
+include ../../../../Makefile.inc
+ciclippy: clippy
```

Comment:
I'm assuming this is the line where I flew too close to the sun and turned on clippy ci for this long before it was ready for it.